book/07-git-tools/sections/credentials.asc

```diff
@@ -19,8 +19,8 @@ Git has a few options provided in the box:
   The downside of this approach is that your passwords are stored in cleartext in a plain file in your home directory.
 * If you're using a Mac, Git comes with an ``osxkeychain'' mode, which caches credentials in the secure keychain that's attached to your system account.
   This method stores the credentials on disk, and they never expire, but they're encrypted with the same system that stores HTTPS certificates and Safari auto-fills.
-* If you're using Windows, you can install a helper called ``wincred.''
-  This is similar to the ``osxkeychain'' helper described above, but uses the Windows Credential Store to control sensitive information.
+* If you're using Windows, you can install a helper called ``Git Credential Manager for Windows.''
+  This is similar to the ``osxkeychain'' helper described above, but uses the Windows Credential Store to control sensitive information. It can be found at https://github.com/Microsoft/Git-Credential-Manager-for-Windows[]. It's the successor to the ``Windows Credential Store for Git'' (``git-credential-winstore'', a.k.a. ``wincred''), which is no longer maintained.
```

Comment:
Two things:

1. This last sentence doesn't seem necessary. There are _some_ places where we talk about how things used to be, but in this case it doesn't really help the reader, since they're most likely looking for current best practices.

2. Can you put each sentence on its own line? This helps keep diffs readable for future PRs.